src/main/java/com/ksm/wstbackoffice/domain/Warehouse.java

```diff
@@ -0,0 +1,27 @@
+package com.ksm.wstbackoffice.domain;
+
+import com.ksm.wstbackoffice.enums.PriceType;
+import com.ksm.wstbackoffice.enums.WarehouseType;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
```

Comment:
Please do not use wildcard import.
Here is more info how to disable it
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports#:~:text=Update%3A%20in%20IDEA%2013%20%22Use,'*'%20to%20a%20higher%20value.

Reply:
done